packages/loot-core/src/server/api.ts

```
@@ -768,6 +775,138 @@
+  const { data } = await aqlQuery(q('schedules').filter({ id }).select('*'));
+  const sched = data[0] as ScheduleEntity;
+  let conditionsUpdated = false as boolean;
+
+  for (const key in fields) {
+    const typedKey = key as keyof APIScheduleEntity;
+    const value = fields[typedKey];
+
+    switch (typedKey) {
+      case 'name':
+        const newName = value as string;
+        const { data } = await aqlQuery(
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)